### DISCUSSION TITLE
Add autocomplete and expression button for the special instruction Repeat

### DISCUSSION THREAD

**Discussion by Bouh** [4ian/GDevelop#3529] [Enhancement ideas (and their technical discussions)]
## Description

Although expressions are permit it's a raw input without indicator and autocomplete.
![image](https://user-images.githubusercontent.com/1670670/128889722-b110f817-9eee-40aa-a6c8-ca94375999fb.png)

## Solution suggested

Add the autocomplete and an icon for select the expressions.

![image](https://user-images.githubusercontent.com/1670670/128889375-6b4e9953-bc26-4bdf-8ceb-16a7e59c9364.png)

**Comment by 4ian** [comment]
Note that the button and selector might not be made to work in a small popover, but we could imagine having a button in the popup to open the edition in a separate, "full screen" (or large) dialog where we can have the entire version of the field (with the button and the selector)
